libraries/ATT_IOT/PayloadBuilder.h

```diff
@@ -28,80 +28,79 @@
 // Data type + size in bytes
 #define ATTALK_BOOLEAN_SIZE 1
 #define ATTALK_INTEGER_SIZE 2
-#define ATTALK_NUMBER_SIZE  4
-#define ATTALK_BYTE_SIZE    1
-#define ATTALK_GPS_SIZE     12
-#define ATTALK_ACCEL_SIZE   12
-
+#define ATTALK_NUMBER_SIZE 4
+#define ATTALK_BYTE_SIZE 1
+#define ATTALK_GPS_SIZE 12
+#define ATTALK_ACCEL_SIZE 12
 
 // AllThingsTalk Payload Builder
-class PayloadBuilder {
-  
-  public:
-  
-    // Constructors
-    PayloadBuilder(ATTDevice &device);
-    PayloadBuilder(ATTDevice &device, uint8_t size);
-    ~PayloadBuilder();
-
-    /**
+class PayloadBuilder
+{
+
+public:
+  // Constructors
+  PayloadBuilder(ATTDevice &device);
+  PayloadBuilder(ATTDevice &device, uint8_t size);
+  ~PayloadBuilder();
+
+  /**
      * Reset the payload, to call before building a frame payload.
      */
-    void reset(void);
-    
-    /**
+  void reset(void);
+
+  /**
      * Send the payload.
      */
-    bool send(void* packet, unsigned char size);
+  bool send(void *packet, unsigned char size);
 
-    /**
+  /**
      * Copy and send the payload.
      */
-    bool send();
-    
-    /**
+  bool send();
+
+  /**
      * Return the current size of the payload.
      */
-    uint8_t getSize(void);
+  uint8_t getSize(void);
 
-    /**
+  /**
      * Return the payload buffer.
      */
-    uint8_t* getBuffer(void);
+  uint8_t *getBuffer(void);
 
-    /**
+  /**
      * Copy the payload buffer.
      */
-    uint8_t copy(uint8_t* buffer);
+  uint8_t copy(uint8_t *buffer);
 
-    /**
+  /**
      * Add a Boolean to the payload buffer.
      *
      * @param value can be 0 or 1, represents the boolean
      *
      * @return size of the payload or 0 when max payload size has been exceeded
      */
-    uint8_t addBoolean(uint8_t value);
+  uint8_t addBoolean(uint8_t value);
 
-    /**
+  /**
      * Add an Integer to the payload buffer.
      *
      * @param value can be a range of -32,768 to 32,767 (2 bytes) value
      *
      * @return length of the payload or 0 when max payload size has been exceeded
      */
-    uint8_t addInteger(int value);
+  uint8_t addInteger(int value);
```

Comment:
So, if the comment above is correct, int could be changed to int16_t?

Reply:
Return value just seems to track a point within the stream. I'm not sure why is it exposed like this though.